### DISCUSSION TITLE
[Feature Request] Tabs in the show view

### DISCUSSION THREAD

**Discussion by soufiene-slimi** [Laravel-Backpack/community-forum#317] [General] [answered]
# Feature Request

### What's the feature you think Backpack should have?

Tabs like the create/edit views

### Have you already implemented a prototype solution, for your own project?

Not yet

### Do you see this as a core feature or an add-on?

Core feature

**Comment by tabacitu** [comment]
Hmm... this might be a good idea @soufiene-slimi - thanks! For large CRUDs one single page is sometimes overwhelming. I'll leave this open to see if more people agree to it.

Cheers!

**Comment by ziming** [comment]
More easy ways to customize the show page is a plus for me :).

**Comment by tabacitu** [comment]
I've moved this to https://github.com/Laravel-Backpack/ideas/issues/1 - let's talk there please.

**Comment by pxpm** [comment, marked as answer]
We have it. https://github.com/Laravel-Backpack/CRUD/pull/5043
